ci_framework/roles/libvirt_manager/templates/kustomize_compute.yml.j2

```diff
@@ -0,0 +1,28 @@
+{%- set compute_range = range(0, compute_config.amount | default(1) | int) | list -%}"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+namespace: {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+patches:
+- target:
+    kind: OpenStackControlPlane
```

Comment:
Kustomize is going to replace these vars using this [cr file](https://github.com/openstack-k8s-operators/install_yamls/blob/master/Makefile#L201)  and the correct[ kind ](https://github.com/openstack-k8s-operators/install_yamls/blob/master/Makefile#L373)is 
```
OpenStackDataPlane
```